nginx/run_nginx.sh

```
@@ -358,10 +360,23 @@ if [[ ! -f /etc/nginx/nginx.conf_orig ]]; then
 		exit_out "Failed to start nginx" $E_GENERAL
 	fi
+	attempts=0
+	while true
+	do
+		systemctl status nginx > /dev/null
+		if [[ $? -eq 0 ]]; then
+			break
+		fi
+		sleep 5
+		if [[ $attempts -eq 12 ]]; then
+			exit_out "nginx never entered the start state" $E_GENERAL
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)